src/pudl/etl/cli.py

```diff
@@ -77,15 +77,6 @@ def get_pudl_etl_job():
     type=int,
     help="Max number of processes Dagster can launch. Defaults to the number of CPUs.",
 )
-@click.option(
-    "--epacems-workers",
-    default=2,
-    type=int,
-    help=(
-        "Max number of processes Dagster can launch for EPA CEMS assets. Defaults "
-        "to max number of processes our typical local machines can handle."
-    ),
-)
```

Comment:
I removed this because I dont' think we are actually changing the number of workers using the command option. We could add an argument that can accept an arbitrary number of asset_name and concurrency limits:

```
pudl_etl src/pudl/package_data/settings/etl_full.yml --tag-concurrency-limits a 1 --tag-concurrency-limits b 2
```

Reply:
In the long term I'd like to just remove the `pudl_etl` command in favor of preconfigured dagster jobs executed using [`dagster job execute`](https://github.com/catalyst-cooperative/pudl/issues/3161) (sorry I still need to flesh out this issue) .

Reply:
Agree that we aren't really using this option, and switching to the dagster native CLI would be good.